eigrpd/eigrp_topology.c

```diff
@@ -448,6 +448,8 @@ void eigrp_topology_update_node_flags(struct eigrp_prefix_entry *dest)
 	struct eigrp_nexthop_entry *entry;
 	struct eigrp *eigrp = eigrp_lookup();
 
+	assert(eigrp);
+
```

Comment:
Strange to see an eigrp modification here as the PR mention only ospfd.

can you clarify ? Split this PR in 2 or change the PR title

Reply:
Thank you, I'll fix it (it was because of grouping two fixes in one).